config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -54,11 +55,12 @@ presubmits:
         - --use-logexporter
         image: gcr.io/k8s-testimages/kubekins-e2e:v20200807-c10eea0-master
         resources:
+          # Using 6 CPU to speed up bazel build phase
```

Comment:
... phase (4 is enough for the test itself).

Same below.

Reply:
Done.